Cubical/HITs/Ints/DeltaInt/Properties.agda

```diff
@@ -41,30 +41,23 @@ cancelDiamond a b i j = hcomp (λ k → λ
 cancelTriangle : ∀ a b i → a ⊖ b ≡ cancel a b i
 cancelTriangle a b i j = hcomp (λ k → λ
   { (i = i0) → a ⊖ b
-  ; (i = i1) → cancel a b (j ∧ k)
   ; (j = i0) → a ⊖ b
   ; (j = i1) → cancel a b (i ∧ k)
   }) (a ⊖ b)
 
-{-
-If you wish to prove this, we wish you good luck.
-
 deltaIntRet : ∀ a → fromInt (toInt a) ≡ a
 deltaIntRet (x ⊖ 0) = refl
 deltaIntRet (0 ⊖ suc y) = refl
 deltaIntRet (suc x ⊖ suc y) = deltaIntRet (x ⊖ y) ∙ cancel x y
--- a ⊖ 0 ≡ cancel a 0 i
-deltaIntRet (cancel a 0 i) = {!cancelTriangle a 0 i!}
-deltaIntRet (cancel 0 (suc b) i) = {!cancelTriangle 0 (suc b) i!}
+deltaIntRet (cancel a 0 i) = cancelTriangle a 0 i
+deltaIntRet (cancel 0 (suc b) i) = cancelTriangle 0 (suc b) i
 deltaIntRet (cancel (suc a) (suc b) i) = deltaIntRet (cancel a b i) ∙ cancelDiamond a b i
 
 DeltaInt≡Int : DeltaInt ≡ Int
 DeltaInt≡Int = isoToPath (iso toInt fromInt deltaIntSec deltaIntRet)
--}
 
 succPred : ∀ n → succ (pred n) ≡ n
 succPred (x ⊖ y) i = cancel x y (~ i)
--- cancel (suc a) (suc b) i ≡ cancel a b i
```

Comment:
Why is this line removed?

Reply:
oh, I cleaned it out accidentally

Reply:
Oh! This is a comment! I didn't notice this. I thought it's one clause of the function. I apologize.